ccd-definition/ComplexTypes/EmploymentTypeComplexTypeLRspec.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "ID": "EmploymentTypeComplexTypeLRspec",
+    "ListElementCode": "employmentType",
+    "ElementLabel": "Employed or self-employed?. \n",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
```

Comment:
This whenWillThisAmountBePaid doesn't seem to belong

Reply:
removed. was not meant to be in this structure